### PR TITLE
goPackages.protobuf: transition to new github import path

### PR DESCRIPTION
**Pull request by benley**
I was thinking of doing this sort of thing as a global goPackages import path transition mechanism, but took the easier approach here for the moment.

[Review on Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/6941)

### CODE REVIEW

**Comment by benley** [comment]
sigh, I wish travis didn't completely faceplant so frequently.

File: pkgs/top-level/go-packages.nix [hunk 1]
```diff
@@ -45,13 +45,14 @@ let self = _self // overrides; _self = with self; {
   };
 
   protobuf = buildGoPackage rec {
-    rev = "36be16571e14";
-    name = "goprotobuf-${rev}";
-    goPackagePath = "code.google.com/p/goprotobuf";
-    src = fetchhg {
+    rev = "5677a0e3d5e89854c9974e1256839ee23f8233ca";
+    name = "protobuf-${rev}";
```
Comment (lethalman) [inline pkgs/top-level/go-packages.nix]: Can we please keep goprotobuf? Because the protobuf name is already used by the protobuf package.

Comment (benley) [inline pkgs/top-level/go-packages.nix]: ack yes of course, I didn't mean to change that. Fixing.

**Comment by lethalman** [comment]
Ok thanks.

**Pull request merged**